### ISSUE TITLE
NamespaceDoc support

### ISSUE THREAD

**Issue by eli-darkly** [praeclarum/FuGetGallery#127]
.NET itself doesn't (as far as I know) have a standard for attaching a documentation comment to a whole namespace, but for a long time the Sandcastle HTML documentation generator has supported doing this by declaring an internal-scoped class called `NamespaceDoc` in that namespace and putting the doc comment on that. Fuget doesn't seem to put any special significance on `NamespaceDoc`, but also, it doesn't seem to even parse the docs for that class correctly as a class: if you click on `NamespaceDoc`, it shows the `<summary>` text but not the `<remarks>`. This means developers may be missing quite a bit of documentation that the library maintainer wrote at the namespace level.

**Comment by praeclarum** [comment]
Yeah I didn't realize how important the remarks were. I will look into adding them in.

I'm a little confused about the NamespaceDoc.  Can you point me to a URL?
